samples/audio-recording-add-on/README.md

```diff
@@ -0,0 +1,16 @@
+## About
+
+This project has been created with _@adobe/create-ccweb-add-on_. As an example, this Add-on demonstrates how to use the Add-on SDK's import audio using the media Recorder api of the browser.
+
+## Technology Used
+
+-   HTML
+-   TypeScript
+-   CSS
+-   Webpack
```

Comment:
There is no webpack and typescript in this add-on.
 Please update this Readme accordingly